src/xdocs/config_annotation.xml

```diff
@@ -159,14 +166,14 @@ public String getNameIfPresent() { ... }
 &lt;/module&gt;
         </source>
         <p>
-          Example to allow one single parameterless annotation on the same line
+          Example to allow multiple annotations on the same line
         </p>
         <source>
-@Override public int hashCode() { ... } // no violations
-@SuppressWarnings("deprecation") public int foo() { ... } // violation
+@SuppressWarnings("deprecation") @Mock DataLoader loader; // no violations
```

Comment:
What was wrong with original examples that they changed here? If its not an issue, we should keep all 3.

We should also add examples to showcases that will cause a violation.

Reply:
Examples on line 165,166 are used for "allowSamelineSingleParameterlessAnnotation", but I'm providing an example for "allowSamelineMultipleAnnotations" on this line. The order has changed since I have added default configuration at the top.